src/main/java/core/basesyntax/FigureGenerator.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureGenerator {
+    private static final int NumberOfFigures = 5;
+    private static final int LimitNumber = 100;
+
+    public static Figure createFigure() {
+        switch (new Random().nextInt(NumberOfFigures)) {
+            case 1:
+                return new Circle(getRandomNumber(), getRandomColor());
+            case 2:
+                return new RightTriangle(getRandomNumber(), getRandomNumber(),
+                        getRandomNumber(), getRandomColor());
+            case 3:
+                return new Rectangle(getRandomNumber(), getRandomNumber(), getRandomColor());
+            case 4:
+                return new Square(getRandomNumber(), getRandomColor());
+            default:
+                return new IsoscelesTrapezoid(getRandomNumber(),
+                        getRandomNumber(), getRandomNumber(), getRandomColor());
+        }
+    }
+
+    private static String getRandomColor() {
+        int randomGenerateIndex = new Random().nextInt(Color.values().length);
+        return Color.values()[randomGenerateIndex].toString();
+    }
```

Comment:
move this functionality in a separate ColorGenerator class
don't make the method static please - let's create a field of ColorGenerator type in this class